website/docs/ocl.md

```diff
@@ -152,16 +170,23 @@ This is the list of security (groups) defined in the service network. Each secur
     }
   },
   "compute": {
-    "vm": [{
-      "name": "my-vm",
-      "type": "t2.large",
-      "platform": "linux-x64",
-      "vpc": "$.network.vpc[0]",
-      "subnet": "$.network.subnet[0]",
-      "security": ["$.network.security[0]"],
-      "storage": ["$.storage[0]}"],
-      "publicly": true
-    }]
+    "vm": [
+      {
+        "name": "my-vm",
+        "type": "t2.large",
+        "platform": "linux-x64",
+        "image": "$.image.artifacts.kafka_image",
+        "vpc": "$.network.vpc[0]",
+        "subnet": "$.network.subnet[0]",
+        "security": [
+          "$.network.security[0]"
+        ],
+        "storage": [
+          "$.storage[0]}"
+        ],
+        "publicly": true
```

Comment:
does this means public ip associated? if so how about replacing with a bandwidth parameter and when it's set greater than 0 means we need a public ip.

Reply:
ok, we can add public ip related things later.